### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by kalidiagne**
**Desktop** 
Screenshot 2023-09-08 at 11 33 55

**Mobile**
Screenshot 2023-09-08 at 11 34 02

### CODE REVIEW

**Comment by kalidiagne** [comment]
There from here 

https://website-v2-git-76-devel-fb72be-privacy-and-scaling-explorations.vercel.app/wwww 

@ChialiT @thebeyondr

File: public/icons/emotion-sad-line.svg
Comment (ChialiT) [inline public/icons/emotion-sad-line.svg:1]: Design team suppose to supply a new asset for this.

**Review by ChialiT — APPROVED** [review]
I will approve as it is, but we need to swap the asset.
once asset is updated, I think we are good to go.

**Review by thebeyondr — CHANGES_REQUESTED** [review]
new asset to replace the sad img

**Comment by kalidiagne** [comment]
> new asset to replace the sad img

Asset replaced 👍

**Review by thebeyondr — APPROVED** [review]

**Pull request merged**